Fix broken PDF build

The PDF build currently fails:

```
[...]

! Package inputenc Error: Unicode character ✘ (U+2718)
(inputenc)                not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.21919 ✘

(That makes 100 errors; please try again.)
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on Mixxx-Manual.log.
=== TeX engine is 'pdfTeX'
Latexmk: Index file 'Mixxx-Manual.idx' was written
Latexmk: Missing input file: 'None' from line
  '! LaTeX Error: File `None' not found.'
Latexmk: Missing input file: 'None' from line
  '! LaTeX Error: File `None' not found.'
Latexmk: Missing input file: 'None' from line
  '! LaTeX Error: File `None' not found.'
Latexmk: Missing input file: 'None' from line
  '! LaTeX Error: File `None' not found.'
Failure to make 'Mixxx-Manual.pdf'
Latexmk: Summary of warnings from last run of (pdf)latex:
  =====Latex reported missing or unavailable character(s).
=====See log file for details.
Latexmk: Errors, in force_mode: so I tried finishing targets
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'Mixxx-Manual.log' for details
make[1]: *** [Makefile:29: Mixxx-Manual.pdf] Error 12
make[1]: Leaving directory '/home/jan/Projects/mixxx-manual/build/latex'
make: *** [Makefile:121: latexpdf] Error 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -37,8 +37,11 @@ jobs:
       addons:
         apt:
           packages:
+            - fonts-freefont-otf
             - texlive
+            - texlive-generic-recommended
             - texlive-xetex
             - texlive-latex-extra
+            - texlive-lang-all
             - latexmk
             - xindy

--- a/build_pdf.sh
+++ b/build_pdf.sh
@@ -1,22 +1,36 @@
+#!/bin/sh
+
 # Parse languages
 cd source || exit 1
+VERSION="$(python -c 'import conf; print(conf.version)')"
 LANGUAGES="$(python -c 'import conf; print(" ".join(sorted((lang for lang in conf.supported_languages.keys()), reverse=True)))')"
 cd .. || exit 1
 NUM_LANGUAGES="$(printf '%s' "$LANGUAGES" | wc -w)"
 
 i=1
+
+mkdir -p build/pdf
 for lang in $LANGUAGES
 do
     printf -- '----- Building language "%s"... [%d/%d] -----\n' "$lang" "$i" "$NUM_LANGUAGES"
-    make versionedlatexpdf SPHINXOPTS="-Q -j $(nproc) -Dlanguage=$lang" >/dev/null
+    SPHINXOPTS="-q -j $(nproc) -Dlanguage=${lang}"
+    LATEXMKOPTS="-f -interaction=nonstopmode -pdf"
+    BUILDDIR="build/latex/${lang}"
+
+    if [ "${lang}" = "ja-JP" ]
+    then
+        SPHINXOPTS="${SPHINXOPTS} -Dlatex_engine=platex"
+    elif [ "${lang}" = "zh-CN" ] || [ "${lang}" = "zh-TW" ]
+    then
+        SPHINXOPTS="${SPHINXOPTS} -Dlatex_engine=xelatex -Dlatex_elements.preamble=\\usepackage[UTF8]{ctex}\\n"
+        LATEXMKOPTS="${LATEXMKOPTS} -xelatex"
+    else
+        SPHINXOPTS="${SPHINXOPTS} -Dlatex_engine=xelatex"
+        LATEXMKOPTS="${LATEXMKOPTS} -xelatex"
+    fi
+
+    sphinx-build -b latex source "${BUILDDIR}" ${SPHINXOPTS}
+    make -C "${BUILDDIR}" LATEXMKOPTS="${LATEXMKOPTS}" all-pdf >/dev/null
+    [ -e "${BUILDDIR}/Mixxx-Manual.pdf" ] && cp "${BUILDDIR}/Mixxx-Manual.pdf" "build/pdf/mixxx-manual-${VERSION}-${lang}.pdf"
     i=$((i + 1))
 done
-
-cd build/latex || exit 1
-mkdir ../pdf
-for file in */*/Mixxx-Manual.pdf
-do
-    newfilename="mixxx-manual-$(dirname "$file" | tr "/" "-")"
-    cp "$file" "../pdf/${newfilename}.pdf"
-done
-find ../pdf -type f

--- a/source/conf.py
+++ b/source/conf.py
@@ -32,6 +32,7 @@ import sphinx_rtd_theme
 extensions = [
     "sphinx.ext.todo",
     "sphinx.ext.graphviz",
+    "sphinx.ext.imgconverter",
     "sphinx_rtd_theme",
     "sphinx_multiversion",
     "sphinx_mixxx",
@@ -284,6 +285,7 @@ latex_elements = {
     "classoptions": ",openany,oneside",
     "babel": "\\usepackage[english]{babel}",
 }
+latex_use_xindy = False
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,

